### PR TITLE
feat: Implemented Phone Auth

### PR DESCRIPTION
**Pull request by mohak1283**
Implemented Phone Authentication and removed Material Design library.

### CODE REVIEW

File: app/src/main/java/com/sekreative/sekreative/ui/auth/AuthFragment.java [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+    ProgressBar progressBar;
+
+    @BindView(R.id.btn_verify)
+    FloatingActionButton verify_button;
+
+    @BindView(R.id.tv_sekreative_label)
+    TextView seKreative_label;
+
+    @BindView(R.id.tv_tnc)
+    TextView tnc_textview;
```
Comment (sriramr98) [inline app/src/main/java/com/sekreative/sekreative/ui/auth/AuthFragment.java]: Use descriptive variable names. Also, use descriptive IDs for views. What does tv_tnc stand for?

File: app/src/main/java/com/sekreative/sekreative/ui/auth/AuthFragment.java [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+        fromTop = AnimationUtils.loadAnimation(getActivity() , R.anim.from_top);
+        phone_edit_text.setAnimation(fromBottom);
+        seKreative_label.setAnimation(fromTop);
+        beCreative_label.setAnimation(fromTop);
+        checkBox.setAnimation(fromBottom);
+        textView15.setAnimation(fromBottom);
+        fab.setAnimation(fromTop);
+        tnc_textview.setAnimation(fromBottom);
+
+        fab.setOnClickListener(new View.OnClickListener() {
```
Comment (sriramr98) [inline app/src/main/java/com/sekreative/sekreative/ui/auth/AuthFragment.java]: Put this function in `onViewCreated` callback. There are certain cases where this will crash. Only initialize variables in this function. Also, move all view initializations to the`onViewCreated` callback. Only have the animation initializations and Firebase initializations here.

File: app/src/main/res/layout/fragment_auth.xml [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+        android:layout_marginTop="20dp"
+        android:text="I am of 13 years or older!"
+        android:textSize="18sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.294"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/et_phone_number" />
+
+    <TextView
+        android:id="@+id/tv_tnc"
```
Comment (sriramr98) [inline app/src/main/res/layout/fragment_auth.xml]: tv_terms is a more descriptive ID, don't you think?

File: app/src/main/res/values/styles.xml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.MaterialComponents.Light.NoActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
```
Comment (sriramr98) [inline app/src/main/res/values/styles.xml]: Why did you remove MaterialComponents? It is default in API 28. Revert it back to MaterialComponents. You do not have to add any dependencies for this.

File: app/src/main/java/com/sekreative/sekreative/ui/auth/AuthFragment.java (hunk 2 above)
Comment (mohak1283) [reply app/src/main/java/com/sekreative/sekreative/ui/auth/AuthFragment.java]: I have to put setOnClickListener method in onViewCreated function?

Comment (mohak1283) [reply app/src/main/java/com/sekreative/sekreative/ui/auth/AuthFragment.java]: And right now, mCallbacks is initialized in onCreateView() so, I have to move that initialization to onViewCreated() ?

File: app/src/main/java/com/sekreative/sekreative/ui/auth/AuthFragment.java (hunk 1 above)
Comment (mohak1283) [reply app/src/main/java/com/sekreative/sekreative/ui/auth/AuthFragment.java]: Done :+1:

File: app/src/main/res/layout/fragment_auth.xml (hunk 3 above)
Comment (mohak1283) [reply app/src/main/res/layout/fragment_auth.xml]: Done :+1:

File: app/src/main/res/values/styles.xml (hunk 4 above)
Comment (mohak1283) [reply app/src/main/res/values/styles.xml]: Done :+1:

**Comment by mohak1283** [comment]
@sriramr98 I have done all the changes requested by you and created updated PR. :+1:

**Comment by mohak1283** [comment]
Thank u @sriramr98 :+1: 
Okay I will remember the naming conventions :+1:
